nixos/modules/system/boot/initrd-network.nix

```diff
@@ -40,6 +40,10 @@ in
         kernel documentation</link>.  Otherwise, if
         <option>networking.useDHCP</option> is enabled, an IP address
         is acquired using DHCP.
+
+        You should add the module(s) required for your network card to
+        boot.initrd.availableKernelModules. lspci -v -s <ethernet controller>
```

Comment:
<ethernet controller> isn't proper markup, this breaks the build.

Reply:
fixed in a1e2f2a3390dcbad439e7cebc8e4fdf4e97f2b20